src/Assets/interaction.js

```diff
@@ -32,13 +32,11 @@ export default class Interaction {
     const checklists = document.getElementById(task.index);
     checklists.addEventListener('change', () => {
       task.completed = !task.completed;
-      //  const descript = document.getElementById(`d${task.index}`);
-      //  descript.classList.toggle('line');
     });
   }
 
   static checkList() {
-    const storage = this.localStorage;
+    const storage = this.localStorageA();
```

Comment:
Oops, we need to address the issue the previous reviewer mentioned.

In other words;

For some reason, our data is not properly saved by the localStorage.

Checked item after reload tends to change to the unchecked item.